apps/ligretto-frontend/src/components/blocks/game/CardHotkeyBadge/CardHotkeyBadge.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { CardHotkeyBadge } from './CardHotkeyBadge'
+import { CardColors } from '@memebattle/ligretto-shared'
+import { CardsRow } from '../CardsRow'
+import { Hotkey } from 'ducks/game'
+
+import { Card } from '../Card'
+
+export default {
+  title: 'Ligretto / CardHotkeyBadge',
+}
+
+export const DefaultView = () => (
+  <CardsRow>
+    <CardHotkeyBadge hotkey={Hotkey.x}>
+      <Card color={CardColors.blue} value={1} />
```

Comment:
Let's wrap Card with CardPlace to make the story be similar to real app component.